posthog/hogql_queries/web_analytics/test/__snapshots__/test_web_overview.ambr

```diff
@@ -12,15 +12,15 @@
 # ---
 # name: TestWebOverviewQueryRunner.test_all_time.1
   '''
-  SELECT uniq(session_person_id) AS `uniq(session_person_id)`,
+  SELECT uniq(session_person_id) AS unique_users,
          NULL AS previous_unique_users,
-         sum(filtered_pageview_count) AS `sum(filtered_pageview_count)`,
-         NULL AS previous_filtered_pageview_count,
-         uniq(session_id) AS `uniq(session_id)`,
+         sum(filtered_pageview_count) AS total_filtered_pageview_count,
+         NULL AS previous_total_filtered_pageview_count,
+         uniq(session_id) AS unique_sessions,
          NULL AS previous_unique_sessions,
-         avg(session_duration) AS `avg(session_duration)`,
+         avg(session_duration) AS avg_duration_s,
          NULL AS prev_avg_duration_s,
-         avg(is_bounce) AS `avg(is_bounce)`,
+         avg(is_bounce) AS bounce_rate,
          NULL AS prev_bounce_rate
```

Comment:
Some are `previous_*` and some are `prev_*`. Can we standardize? I'm fine with `prev`